lib/ex_doc/formatter/html/autolink.ex

```diff
@@ -721,6 +752,12 @@ defmodule ExDoc.Formatter.HTML.Autolink do
     }x
   end
 
+  defp re_kind_language(:mix_task, :elixir) do
+    ~r{
+      mix\ ([a-z]+[a-z0-9\._]*)
```

Comment:
Should we handle more than one white-space?

Reply:
I'd keep it as is, if someone accidentally adds more it won't highlight so it's gonna be easier to spot.

Reply:
My only concern is that if a tests editor wraps the line and we end up with
```
`mix
task argument`
```

or

```
`mix task
arguments`
```

Reply:
@eksperimental i think we can wait until it happens.

Reply:
> @eksperimental i think we can wait until it happens.

already happening
https://github.com/elixir-lang/elixir/pull/8372/files#diff-c12d92120ee0fe3d324a448c404f75a6L26